smoker.spec

```diff
@@ -0,0 +1,58 @@
+%global branch master
+%global candidate rc2
```

Comment:
It would be best to generate spec file somehow that having version on multiple places.